src/pages/QuestionsPage.tsx

```diff
@@ -67,20 +91,18 @@ const QuestionsPage: React.FC = () => {
 
   return (
     <ListPage>
-      {me && (
-        <ConversationsMessage
-          onClick={() => {
-            history.push('/conversations/onboarding')
-          }}
-        >
-          Hey there <Capitalized>{getNameFromEmail(me.email)}</Capitalized>
-          !
-          <br />
-          <span style={{ fontSize: '0.9em' }}>
-            We're testing a new version of the question page, want to try it?
-          </span>
-        </ConversationsMessage>
-      )}
+      <ConversationsMessage
+        onClick={() => {
+          history.push('/conversations/onboarding')
+        }}
+      >
+        Hey there <Capitalized>{getNameFromEmail(me.email)}</Capitalized>
```

Comment:
```suggestion
        Hey there <Capitalized>{me.fullName}</Capitalized>
```

Or just extract the first name from the `fullName` as yesterday 😄